src/__tests__/App.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import App from '../App';
+import store from '../redux/configurestore';
+
+describe('App', () => {
+  it('renders correctly', () => {
+    const tree = renderer.create(<Provider store={store}><App /></Provider>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Good job adding integration tests to your application! These are great to make sure that everything is working as expected. Let's update the snapshots since the last time they were generated they have been changed and are currently failing.

![image](https://user-images.githubusercontent.com/79114233/142741609-a11e59cb-c46b-4920-aa75-d413e492c9af.png)